packages/sync/proguard-rules-consumer-common.pro

```diff
@@ -0,0 +1,20 @@
+# Keep all classes implemeting the RealmObject interface
+-keep class io.realm.RealmObject
+-keep class * implements io.realm.RealmObject { *; }
+#-keep class **.$* implements io.realm.RealmObject { *; }
+
+# Preserve all native method names and the names of their classes.
+-keepclasseswithmembernames,includedescriptorclasses class * {
+    native <methods>;
+}
+
+# Notification callback
+-keep class io.realm.interop.NotificationCallback {
```

Comment:
Do we need these here in the sync variant? I assumed they would be inherited?

Reply:
True, forgot about proguard cleanup.